templates/firefox-ios.xliff

```diff
@@ -1251,6 +1288,14 @@
         <source>Tracking Protection is now on for this site.</source>
         <note>The confirmation toast once tracking protection has been enabled</note>
       </trans-unit>
+      <trans-unit id="Menu.TrackingProtectionFingerprintersBlocked.Title">
+        <source>Fingerprinters</source>
+        <note>The title that shows the number of fingerprinting scripts blocked</note>
+      </trans-unit>
+      <trans-unit id="Menu.TrackingProtectionListTitle">
+        <source>Blocked %@</source>
+        <note>Title for list of domains blocked by category.</note>
```

Comment:
It's a bit confusing to me what  '%@' refers to here. Could you please explicitly call out what the placeholder will be replaced with, in the localization comment?
(probably good to do the same with the other instance in string 1227, even if for me it's less confusing)